packages/twenty-front/src/modules/views/hooks/useSaveCurrentViewGroups.ts

```diff
@@ -20,6 +20,56 @@ export const useSaveCurrentViewGroups = (viewBarComponentId?: string) => {
     viewBarComponentId,
   );
 
+  const saveViewGroup = useRecoilCallback(
+    ({ snapshot }) =>
+      async (viewGroupToSave: ViewGroup) => {
```

Comment:
views hooks should expose RecordGroups in its API and internally convert it to view Group, this way everybody is taking dependency on RecordGroup (types) and ViewGroups is only in ViewBar

Reply:
I'll take a look on tha

Reply:
yes, please discuss with Lucas too on this